<a href="https://colab.research.google.com/github/jmorral/Salary-calculation-function-python/blob/main/Summer_Salary_Function.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Summer Salary Worksheet

***Please note that the function will need to be updated if overhead and fringe rates change for the different award years. 

In [ ]:
#@title
def summer_salary(salary, fund_type, months):
  """Calculate the full cost of summer salary for a professor taking fund type into consideration."""
  fringe_unrestricted = 1.30
  fringe_restricted = 1.30
  overhead = 1.585
  salary = salary/9
  return (salary * fringe_restricted * overhead * months) if fund_type == 25 or fund_type == 24 else (salary * fringe_unrestricted * months)

This worksheet can be easily used to determine a professor's summer salary. 

This is how to use this function:

        function(arguments)

        summer_salary(salary, fund_type, months)

So for a professor with a salary of $100,000 a year, paid by a fund 25, for 2 months, you would enter: 

        summer_salary(100000, 25, 2) 


Open a new code cell and try:

In [ ]:
summer_salary(100000, 25, 2)

45788.88888888889

The returned value of $45,788.89 is for two months of summer salary on a fund 25 grant!
